lib/http_adapter/node.js

```diff
@@ -14,6 +14,12 @@ Http.prototype.post = function(req) {
         headers['x-tagged-client-id'] = req.clientId;
     }
 
+    if (req.secret) {
+        headers['x-tagged-client-secret'] = req.secret;
+    }
+
+    console.info('post /api headers: ', headers);
```

Comment:
This will be removed.  Still doing some testing for now to make sure it works properly.